legion/engine/core/serialization/serialization.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
```

Comment:
No documentation in this file at __all__

Reply:
docs is for a later subfeature

Reply:
seems odd but alright, since it's a subfeature that's acceptable I suppose